### PR TITLE
Remove all model references

### PR DESCRIPTION
**Pull request by phuclh**
- Remove all model references in the config file.
- Replace model references with new alias from support package.

### CODE REVIEW

**Review by drewroberts — APPROVED** [review]
Thank you, Phuc! 🔌

File: src/Models/Rate.php [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        return $this->belongsTo(config('tipoff.model_class.user'), 'creator_id');
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function updater()
+    {
+        return $this->belongsTo(config('tipoff.model_class.user'), 'updater_id');
+    }
```
Comment (drewroberts) [inline src/Models/Rate.php:149]: I'm going to pull this out in all the packages to a HasUpdater trait in the support package.

File: src/Models/Rate.php [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        static::creating(function ($rate) {
+            if (auth()->check()) {
+                $rate->creator_id = auth()->id();
+            }
+        });
+
+        static::saving(function ($rate) {
+            if (auth()->check()) {
+                $rate->updater_id = auth()->id();
+            }
```
Comment (drewroberts) [inline src/Models/Rate.php:27]: I'm going to pull this out in all the packages to a HasUpdater trait in the support package.

File: src/Models/Rate.php [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+    protected static function boot()
+    {
+        parent::boot();
+
+        static::creating(function ($rate) {
+            if (auth()->check()) {
+                $rate->creator_id = auth()->id();
+            }
+        });
```
Comment (drewroberts) [inline src/Models/Rate.php:22]: I'm going to pull this out in all the packages to a HasCreator trait in the support package.

File: src/Models/Rate.php [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+        return $this->$key * $participants;
+    }
+
+    /**
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function creator()
+    {
+        return $this->belongsTo(config('tipoff.model_class.user'), 'creator_id');
+    }
```
Comment (drewroberts) [inline src/Models/Rate.php:141]: I'm going to pull this out in all the packages to a HasCreator trait in the support package.

File: src/Models/Rate.php (hunk 3 above)
Comment (pdbreen) [reply src/Models/Rate.php:22]: If I recall properly, there's a trick to using trait methods in boot hooks.  (Example - timestamp support isn't done in the hooks)

**Pull request merged**